common:
    lid_topic: "/livox/lidar"
    imu_topic: "/livox/imu"
    reloc_topic: "/reloc_location"
    time_sync_en: false # ONLY turn on when external time synchronization is really not possible
    time_offset_lidar_to_imu:
        0.0 # Time offset between lidar and IMU calibrated by other algorithms, e.g. LI-Init (can be found in README).
        # This param will take effect no matter what time_sync_en is. So if the time offset is not known exactly, please set as 0.0

preprocess:
    lidar_type: 1 # 1 for Livox serials LiDAR, 2 for Velodyne LiDAR, 3 for ouster LiDAR,
    scan_line: 6
    blind: 4
    point_filter_num: 3

mapping:
    acc_cov: 0.1
    gyr_cov: 0.1
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    fov_degree: 90
    det_range: 450.0
    extrinsic_est_en: false # true: enable the online estimation of IMU-LiDAR extrinsic
    extrinsic_T: [0.04165, 0.02326, -0.0284]
    extrinsic_R: [1, 0, 0, 0, 1, 0, 0, 0, 1]

publish:
    path_en: true
    scan_publish_en: true # false: close all the point cloud output
    dense_publish_en: true # false: low down the points number in a global-frame point clouds scan.
    scan_bodyframe_pub_en: true # true: output the point cloud scans in IMU-body-frame

init_method: PPICP
NDT:
ICP:
    max_iter: 10
PPICP:
    plane_dist: 0.1
    max_iter: 10
prior_T: [14.5, -7.5, 0]
